frontend/src/stimulus/controllers/dynamic/overview/add-widgets.controller.ts

```diff
@@ -0,0 +1,9 @@
+import { Controller } from '@hotwired/stimulus';
+
+export default class AddWidgetsController extends Controller {
+  private addWidget():void {
+    document.dispatchEvent(
+      new CustomEvent('overview:addWidget'),
+    );
+  }
```

Comment:
Stimulus Controllers provide a `dispatch` method that we should be able to use here.

```suggestion
  private addWidget():void {
    this.dispatch('addWidget');
  }
```

Docs here: https://stimulus.hotwired.dev/reference/controllers#cross-controller-coordination-with-events